packages/components/text/src/text.ts

```diff
@@ -0,0 +1,39 @@
+import { buildProps } from '@element-plus/utils'
+import { componentSizes } from '@element-plus/constants'
+import type Text from './text.vue'
+
+import type { ExtractPropTypes } from 'vue'
+
+export const textProps = buildProps({
+  /**
+   * @description text type
+   */
+  type: {
+    type: String,
+    values: ['primary', 'success', 'info', 'warning', 'danger', ''],
+    default: '',
+  },
+  /**
+   * @description text size
+   */
+  size: {
+    type: String,
+    values: componentSizes,
+    default: '',
+  },
+  /**
+   * @description render ellipsis
+   */
+  truncated: {
+    type: Boolean,
+  },
+  /**
+   * @description custom element tag
+   */
+  tag: {
+    type: String,
```

Comment:
## Suggest

Inspired by the link below, it would be nice to improve the tag prop to receive components.

You can do this later!

https://github.com/element-plus/element-plus/pull/11853/files#diff-043025f7e5f7add14be78f47786cc3980c374f9e6d7ba5f02fc9b901b2d5d1cfR110

Reply:
The `tag` attribute of the Button component supports incoming components mainly to support `router-link` and `nuxt-link`. This component is not recommended this.

Reply:
I see 👌